include/myst/thread.h

```diff
@@ -5,6 +5,7 @@
 #define _MYST_THREAD_H
```

Comment:
Please don't use this technique to work around circular dependencies. I think there is a better way to resolve.

Reply:
Please disregard. I am concerned about circular inclusions elsewhere though.